datalad/support/sshconnector.py

```diff
@@ -111,15 +112,16 @@ def open(self):
         connection, if it is not there already.
         """
 
-        # set control options
-        ctrl_options = ["-o", "ControlMaster=auto", "-o", "ControlPersist=yes"] + self.ctrl_options
-        # create ssh control master command
-        cmd = ["ssh"] + ctrl_options + [self.host, "exit"]
+        if not exists(self.ctrl_path):
```

Comment:
I don't think so. But we probably should ask the ssh executable anyway indeed. Will figure it out.